### ISSUE TITLE
Evaluate Material Admin for administrative theme

### ISSUE THREAD

**Issue by adamdelaney** [uiowa/sitenow#68]
This was mentioned as a modern administrative UI for d8

**Comment by adamdelaney** [comment]
My initial thought is that Material Admin boasts very clean and modern looking interface.

**Comment by adamdelaney** [comment]
We will come back to this after we have our service fleshed out better.

**Comment by joewhitsitt** [comment]
I know it is postponsed, but I tried it out with bootstrap_paragraphs. Below is a comparison of the same form (material/adminimal). My vote is adminimal.

Notice the buttons don't have text... I deleted a paragraph item and it was not very intuitive that I should click the button again to "Confirm deletion."

![edit-material](https://user-images.githubusercontent.com/4663676/40139523-be40c7ae-5915-11e8-9893-85d4a146b28f.png)

![edit-adminimal](https://user-images.githubusercontent.com/4663676/40139522-be2e9c00-5915-11e8-8a30-472b3c11610f.png)
